indico/modules/categories/templates/display/category.html

```diff
@@ -6,6 +6,11 @@
 
 {% block content %}
     <div class="category-info">
+        {% if category.has_logo %}
+            <div class="category-logo">
+                <img src="{{ category.logo_url }}" alt="{% trans %}Logo of the category{% endtrans %}">
```

Comment:
I don't think this alt text is very useful. an empty string makes lots of sense here since the logo has no meaning if the image is not visible for some reason